Zoom Buttons/ZoomButtons.js

```diff
@@ -0,0 +1,201 @@
+
+        });
+
+        var checkZoomButton = function(){
+
+            var was_open = false;
+            if ( isActionMenuOpen() ) {
+                was_open = true;
+            }
+
+            // console.log( "checked: ", was_open );
+
+            if ( !was_open ) openActionMenu();
+
+            if ( $('.ui-slider--zoomer').length ) {
+                var zoom = $('.ui-slider--zoomer').slider('value');                     // read zoom from slider
+                setZoomHighlight( zoom );
+            }
+            if ( !was_open ) closeActionMenu();
+
+            // try again when menu is open
+            if ( was_open ) {
+                zoomTimer = setTimeout(checkZoomButton, 1000);  // try again, slider was not ready
+            }
+        };
+
+        var waitForSlider = function( tries ) {
+
+            openActionMenu();
+
+            // console.log( 'wait before');
+
+            if ( $('button.tau-btn.tau-btn--view-switch.i-role-board-tooltip.tau-btn-list-view.tau-checked').length ) {
+                // list view, just close and skip
+                // hide buttons ?
+                $('#zoom-buttons').hide();
+                closeActionMenu();
+                return false;
+            }
+
+
+
+            if ( $('.ui-slider--zoomer').length ) {
+                // console.log( 'wait inner');
+
+                var zoom = $('.ui-slider--zoomer').slider('value');                     // read zoom from slider
+                setZoomHighlight( zoom );
+
+
+                closeActionMenu();
+
+                // start polling sync when menu is toggled, only open, only if triggered manually
+                $('.tau-board-header__control--actions .tau-actions-btn:not(.tau-checked)').on('click', function() {
+                    // console.log('toggle menu');
+                    if( !$(this).hasClass('zoom-open') ) {
+                        // console.log('start poll');
+                        zoomTimer = setTimeout(checkZoomButton, 1000);  // test if slider has changed
+                    } else {
+                        // console.log('no poll');
+                    }
+
+                    return true;
+                });
+
+
+                return true;
+            } else {
+                // console.log('menu is ok');
+                if ( tries > 0 ) {
+                    setTimeout( function() { waitForSlider( tries -1 ) }, 1000 );  // try again, slider was not ready
+                } else {
+                    // give up , to many tries without menu
+                    closeActionMenu();
+                }
+
+                return false;
+            }
+        };
+
+
+
+        var isActionMenuOpen = function() {
+            return $('.tau-board-header__control--actions .tau-actions-btn.tau-checked').length;
+        };
+
+        var openActionMenu = function() {
+            $('.tau-board-header__control--actions .tau-actions-btn:not(.tau-checked)').addClass('zoom-open'); // mark
```

Comment:
`Actions` menu blinks in IE.

Can you, please, try to hide/z-index menu when open and restore after close to prevent blinking?